lib/tutorials/en_US/caching.md

```diff
@@ -108,20 +108,20 @@ hapi initializes a default [client](https://github.com/hapijs/catbox#client) usi
 ```javascript
 'use strict';
 
-const Hapi = require('hapi');
+const Hapi = require('@hapi/hapi');
 
 const server = Hapi.server({
     port: 8000,
     cache: [
         {
             name: 'mongoCache',
-            engine: require('catbox-mongodb'),
+            engine: require('@hapi/catbox-mongodb'),
```

Comment:
this one doesn't exist: https://www.npmjs.com/package/@hapi/catbox-mongodb

Reply:
Ahh, ok. I'm on it

Reply:
The repo has been archived apparently on GitHub: https://github.com/outmoded/catbox-mongodb. I guess it is not maintained anymore, isn't it?

Reply:
Doesn't look like it. I'll just remove it and keep catbox-redis.

Reply:
Nice

Reply:
forgot to lint....redo!